### PR TITLE
[TA] Modify wording on samples

### PR DESCRIPTION
**Pull request by maririos**
Giving some love to the Analyze samples.

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 1]
```diff
@@ -530,7 +530,7 @@ The Analyze functionality allows to choose which of the supported Text Analytics
         DisplayName = "AnalyzeOperationSample"
     };
 
-    AnalyzeBatchActionsOperation operation = await client.StartAnalyzeBatchActionsAsync(batchDocuments, batchActions);
+    AnalyzeBatchActionsOperation operation = await client.StartAnalyzeBatchActionsAsync(batchDocuments, actions);
 
     await operation.WaitForCompletionAsync();
```
Comment (suhas92) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:536]: Should we add operation metadata like expired on, created on etc?

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 2]
```diff
@@ -493,8 +493,8 @@ Text Analytics for health is a containerized service that extracts and labels re
 }
 ```
 
-### Run Analyze Operation Asynchronously
-The Analyze functionality allows to choose which of the supported Text Analytics features to execute in the same set of documents. Currently the supported features are: entity recognition, linked entity recognition, key phrase extraction, and Personally Identifiable Information (PII) Recognition. For more information see [How to: Use Text Analytics for analyze operation][analyze_operation_howto].
+### Run multiple actions Asynchronously
+This functionality allows to run multiple actions in one or more documents. Actions include entity recognition, linked entity recognition, key phrase extraction, and Personally Identifiable Information (PII) Recognition.. For more information see [Using analyze][analyze_operation_howto].
```
Comment (suhas92) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:497]:
```suggestion
This functionality allows running multiple actions in one or more documents. Actions include entity recognition, linked entity recognition, key phrase extraction, and Personally Identifiable Information (PII) Recognition. For more information see [Using analyze][analyze_operation_howto].
```

**Review by suhas92 — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 1 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/README.md:536]: Added

**Comment by maririos** [comment]
FYI the nightly build is broken so the net - textanalytics - tests will have failures. I still triggered because I want to validate my changes in the samples and those, should be green

**Comment by maririos** [comment]
As expected, failures are in healthcare because of new model version deployed by the service.
This doesn't affect this PR, so merging

**Pull request merged**